### ISSUE TITLE
[vscode] robot args option

### ISSUE THREAD

**Issue by DetachHead** [robocorp/robotframework-lsp#439]
**Is your feature request related to a problem? Please describe.**
i would like to be able to specify additional arguments such as `--listener` that are always passed to robot when clicking either "Run" or "Debug"
![image](https://user-images.githubusercontent.com/57028336/131758376-f742fffb-5920-49b7-9c42-545255d97ec8.png)

**Describe the solution you'd like**
an option to specify args passed to `robot` when running tests

**Describe alternatives you've considered**
#433

**Comment by Lomomo** [comment]
@DetachHead 
You can refer to my launch.json config and change args to what you need. It can pass arguments to robot command when you click "Run" button. In my case, it works.
![image](https://user-images.githubusercontent.com/21312157/131771650-b63183ae-d0f6-422d-8976-7d67e22e0f42.png)

**Comment by DetachHead** [comment]
yeah i know you can specify args in the debug config but it would be nice if you could specify args that are used when you click the inline run/debug buttons
![image](https://user-images.githubusercontent.com/57028336/131774195-5c8d064c-6eff-4bdb-a5e2-bd0a674a5a4f.png)

**Comment by Lomomo** [comment]
@DetachHead  
> yeah i know you can specify args in the debug config but it would be nice if you could specify args that are used when you click the inline run/debug buttons

Yes, the approach that I said is works for the inline "run" button.

**Comment by DetachHead** [comment]
oh i see, it's the name `"Robot Framework: Launch template"` that does it. thanks!

though i think it would be better if this was defined in the settings rather than by using the name of a debug config, which is something that shouldn't have semantic meaning imo
